packages/eslint-plugin/src/rules/space-infix-ops.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  AST_TOKEN_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import baseRule from 'eslint/lib/rules/space-infix-ops';
+import * as util from '../util';
+
+export type Options = util.InferOptionsTypeFromRule<typeof baseRule>;
+export type MessageIds = util.InferMessageIdsTypeFromRule<typeof baseRule>;
+
+export default util.createRule<Options, MessageIds>({
+  name: 'space-infix-ops',
+  meta: {
+    type: 'layout',
+    docs: {
+      description:
+        'This rule is aimed at ensuring there are spaces around infix operators.',
+      category: 'Stylistic Issues',
+      recommended: false,
+      extendsBaseRule: true,
+    },
+    fixable: baseRule.meta.fixable,
+    schema: baseRule.meta.schema,
+    messages: baseRule.meta.messages,
```

Comment:
The `messages` for this rule were only added in v7, so we need this fallback for older versions of ESLint, or else the rule will crash.

```suggestion
    messages: baseRule.meta.messages ?? {
      missingSpace: "Operator '{{operator}}' must be spaced."
    },
```

Reply:
Fixed it! Thanks for the review. It was a first with me and i liked it, i'll definitely try to contribute again